opm/parser/eclipse/EclipseState/Grid/GridProperties.hpp

```diff
@@ -124,6 +146,17 @@ namespace Opm {
         void handleCOPYREGRecord( const DeckRecord& record, const GridProperty<int>& regionProperty );
 
 
+        struct iterator : public storageIteratorType {
+            iterator( storageIteratorType iter ) : storageIteratorType( iter ) { }
```

Comment:
Is this even necessary? Anyway, `using storageIterator::storageIterator` accomplishes the same.